rpcs3/Emu/RSX/RSXThread.cpp

```diff
@@ -347,6 +347,17 @@ namespace rsx
 		if (supports_native_ui)
 		{
 			m_overlay_manager = fxm::make_always<rsx::overlays::display_manager>();
+
+			if (g_cfg.video.perf_overlay.perf_overlay_enabled)
+			{
+				auto perf_overlay = std::make_unique<rsx::overlays::perf_metrics_overlay>(false);
+				perf_overlay->set_detail_level(g_cfg.video.perf_overlay.level);
```

Comment:
You can also just have `auto perf_overlay = m_overlay_manager->create<rsx::overlays::perf_metric_overlay>(false)` to avoid the add later on. Its just a nit though as this is functionally equivalent.

Reply:
Yeah but I needed to call all these setters afterwards, seems better this way than calling `.get()` or whatever on the display manager.